src/styles/globals.css

```diff
@@ -26,3 +26,175 @@
   margin: 0;
   padding: 0;
 }
+
+/* mobile */
+.large-title {
+  font-size: 30px;
+  line-height: 40px;
+}
+
+.title-1 {
+  font-size: 27px;
+  line-height: 33px;
+}
+
+.title-2 {
+  font-size: 21px;
+  line-height: 26px;
+}
+
+.title-3 {
+  font-size: 19px;
+  line-height: 24px;
+}
+
+.headline {
+  font-size: 16px;
+  line-height: 21px;
+}
+
+.body {
+  font-size: 16px;
+  line-height: 21px;
+}
+
+.callout {
+  font-size: 15px;
+  line-height: 20px;
+}
+
+.subhead {
+  font-size: 14px;
+  line-height: 19px;
+}
+
+.footnote {
+  font-size: 12px;
+  line-height: 16px;
+}
+
+.caption-1 {
+  font-size: 11px;
+  line-height: 13px;
+}
+
+.caption-2 {
+  font-size: 11px;
+  line-height: 13px;
+}
+
+/* Tablet */
+@media (min-width: 768px) {
+  .large-title {
+    font-size: 34px;
+    line-height: 41px;
+  }
+
+  .title-1 {
+    font-size: 28px;
+    line-height: 34px;
+  }
+
+  .title-2 {
+    font-size: 22px;
+    line-height: 28px;
+  }
+
+  .title-3 {
+    font-size: 20px;
+    line-height: 25px;
+  }
+
+  .headline {
+    font-size: 17px;
+    line-height: 22px;
+  }
+
+  .body {
+    font-size: 17px;
+    line-height: 22px;
+  }
+
+  .callout {
+    font-size: 16px;
+    line-height: 21px;
+  }
+
+  .subhead {
+    font-size: 15px;
+    line-height: 20px;
+  }
+
+  .footnote {
+    font-size: 13px;
+    line-height: 18px;
+  }
+
+  .caption-1 {
+    font-size: 12px;
+    line-height: 16px;
+  }
+
+  .caption-2 {
+    font-size: 11px;
+    line-height: 13px;
+  }
+}
+
+/* Desktop */
+@media (min-width: 1024px) {
+  .large-title {
+    font-size: 36px;
+    line-height: 43px;
+  }
+
+  .title-1 {
+    font-size: 30px;
+    line-height: 37px;
+  }
+
+  .title-2 {
+    font-size: 24px;
+    line-height: 30x;
+  }
+
+  .title-3 {
+    font-size: 22px;
+    line-height: 28px;
+  }
+
+  .headline {
+    font-size: 19px;
+    line-height: 24px;
+  }
+
+  .body {
+    font-size: 19px;
+    line-height: 24px;
+  }
+
+  .callout {
+    font-size: 18px;
+    line-height: 23px;
+  }
+
+  .subhead {
+    font-size: 17px;
+    line-height: 22px;
+  }
+
+  .footnote {
+    font-size: 15px;
+    line-height: 20px;
+  }
+
+  .caption-1 {
+    font-size: 14x;
+    line-height: 19px;
+  }
+
+  .caption-2 {
+    font-size: 13px;
+    line-height: 18px;
+  }
+}
```

Comment:
Personally, I prefer overwriting tailwind theme than this way. 
However, I think we can move on with this atm.